### ISSUE TITLE
Adapt final API in CIP-0381

### ISSUE THREAD

**Issue by iquerejeta** [cardano-foundation/CIPs#475]
Adapt final changes of the API of the BLS12-381 bindings. For the moment, these are:
* Points are guaranteed to be in the subgroup
* pairing renamed to `millerLoop`
* output of the latter is `MLResult`
* Include `PT` points and it's deserialisation results.

**Comment by iquerejeta** [comment]
Yes, I will open the PR 👌  We are currently working on some final changes. Wanted to assign myself, but couldn't do it.

**Comment by rphair** [comment]
@iquerejeta I believe it was just a record-keeping problem (due to not being linked through a GitHub comment) that caused this issue not to be closed by this PR (please correct me if I'm wrong):
* https://github.com/cardano-foundation/CIPs/pull/506
